From 6c8d28a0fbc2b47d97f56a0503b299d3b933d011 Mon Sep 17 00:00:00 2001
From: Peng Ma <peng.ma@nxp.com>
Date: Mon, 24 Jun 2019 17:21:51 +0800
Subject: [PATCH 549/741] arm64: dts: fsl: ls1028a: Fixed qdma node to fit
 driver.

commit a528d582aab676a5899a6fcab1613b7f53aff9b4 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Because the lsdk qDMA driver is not update with upstream, we should use
the old qdma node to fit for driver probe.
This patch is to fix qdma node doesn't fit for the qdma driver.

Signed-off-by: Peng Ma <peng.ma@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index af576d1e3c37..a09911e88e8a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -652,10 +652,10 @@
 				     <GIC_SPI 254 IRQ_TYPE_LEVEL_HIGH>;
 			interrupt-names = "qdma-error", "qdma-queue0",
 				"qdma-queue1", "qdma-queue2", "qdma-queue3";
-			dma-channels = <8>;
+			channels = <8>;
 			block-number = <1>;
 			block-offset = <0x10000>;
-			fsl,dma-queues = <2>;
+			queues = <2>;
 			status-sizes = <64>;
 			queue-sizes = <64 64>;
 		};
-- 
2.17.1

